pkg/utils/parse_arf_result.go

```diff
@@ -15,17 +19,19 @@ import (
 )
 
 const (
-	machineConfigFixType         = "urn:xccdf:fix:script:ignition"
-	kubernetesFixType            = "urn:xccdf:fix:script:kubernetes"
-	ocilCheckType                = "http://scap.nist.gov/schema/ocil/2"
-	rulePrefix                   = "xccdf_org.ssgproject.content_rule_"
-	questionnaireSuffix          = "_ocil:questionnaire:1"
-	questionSuffix               = "_question:question:1"
-	dependencyAnnotationKey      = "complianceascode.io/depends-on"
-	kubeDependencyAnnotationKey  = "complianceascode.io/depends-on-obj"
-	remediationTypeAnnotationKey = "complianceascode.io/remediation-type"
-	enforcementTypeAnnotationKey = "complianceascode.io/enforcement-type"
-	optionalAnnotationKey        = "complianceascode.io/optional"
+	machineConfigFixType            = "urn:xccdf:fix:script:ignition"
+	kubernetesFixType               = "urn:xccdf:fix:script:kubernetes"
+	ocilCheckType                   = "http://scap.nist.gov/schema/ocil/2"
+	rulePrefix                      = "xccdf_org.ssgproject.content_rule_"
+	valuePrefix                     = "xccdf_org.ssgproject.content_value_"
+	questionnaireSuffix             = "_ocil:questionnaire:1"
+	questionSuffix                  = "_question:question:1"
+	dependencyAnnotationKey         = "complianceascode.io/depends-on"
+	kubeDependencyAnnotationKey     = "complianceascode.io/depends-on-obj"
+	remediationTypeAnnotationKey    = "complianceascode.io/remediation-type"
+	enforcementTypeAnnotationKey    = "complianceascode.io/enforcement-type"
+	optionalAnnotationKey           = "complianceascode.io/optional"
+	valueInputRequiredAnnotationKey = "complianceascode.io/value-input-reqired"
```

Comment:
typo of `required` in the annotation string

Reply:
Hi, Matt, thanks for the review, I have corrected that, I also added a unit test to test the `complianceascode.io/value-input-reqired`.